packages/atomic/src/components/search/atomic-ipx-layout/atomic-ipx-layout.pcss

```diff
@@ -0,0 +1,45 @@
+/**
+ * @prop --atomic-ipx-width: Width of the IPX
+ * @prop --atomic-ipx-height: Height of the IPX
+ */
+atomic-ipx-layout {
+  display: grid;
+  grid-template-rows: auto auto auto 8fr 1fr;
+  max-height: 100%;
+  box-sizing: border-box;
+  width: var(--atomic-ipx-width, 500px);
+  height: var(--atomic-ipx-width, 1000px);
```

Comment:
`rem` scales with the `font-size` defined in the `html` tag at the root of the document, which can be customized in the browser's settings or simply with the browser's dev tools by adding `style=` to the `html` tag.

Since all our components use `rem`, I would recommend testing how much we can increase the font size with the UI still looking good. If we can't increase it that much, we may want to use `rem` here too.

Reply:
@dguerinCoveo devrais pas être `--atomic-ipx-height` ici ?